pkg/router/template/certmanager.go

```diff
@@ -12,10 +12,25 @@ import (
 	routeapi "github.com/openshift/origin/pkg/route/apis/route"
 )
 
+// certificateFile represents a certificate file.
+type certificateFile struct {
+	CertDir  string
+	ID       string
+	Contents []byte
```

Comment:
Is there any reason to make these fields public (upper-case initial letter)?

Reply:
No particular reason (well other than did have it originally in `types.go` with an uppercase `CertificateFile` name) - will change to lower case.